### PR TITLE
Documentation Updates To Help Contributors

### PR DESCRIPTION
**Pull request by ashcrow**
- Added todos.
- Updated cloud-init with new location in tools.
- Moved in the cloud-init code.
- Update authentication_devel to current state.

### CODE REVIEW

File: doc/authentication_devel.rst [hunk 1]
```diff
@@ -18,6 +18,40 @@ override ``__init__`` adding in keyword arguments.
...
+            :param start_response: WSGI start response callable.
+            :type start_response: callable
+            :returns: True on success, False on failure
+            :rtype: bool
+            """
+            if environ.get('wsgi.url_scheme', 'http') == 'https':
+                return True
+            return False
+
+    #: Alias AllwaysAllowOnSSL
```
Comment (cooktheryan) [inline doc/authentication_devel.rst]: Nit AlwaysAllowOnSSL

File: doc/authentication_devel.rst [hunk 2]
```diff
@@ -18,6 +18,40 @@ override ``__init__`` adding in keyword arguments.
...
+````````
+
+Basic
+~~~~~
+
+.. code-block:: python
+
+    from commissaire_http.authentication import Authenticator
+
+    class AllwaysAllowOnSSL(Authenticator):
```
Comment (cooktheryan) [inline doc/authentication_devel.rst]: Same as below

File: doc/authentication_devel.rst (hunk 1 above)
Comment (ashcrow) [reply doc/authentication_devel.rst]: :face_with_head_bandage: :palm_tree:

File: doc/authentication_devel.rst (hunk 2 above)
Comment (ashcrow) [reply doc/authentication_devel.rst]: Fixed both in the fixup.

**Review by cooktheryan — APPROVED** [review]

**Comment by ashcrow** [comment]
[(2) [multi-host-mgr-2] Add todo to docs](https://trello.com/c/bMJczpe8/513-2-multi-host-mgr-2-add-todo-to-docs)

**Pull request merged**